pyvo/dal/sia2.py

```diff
@@ -884,7 +885,11 @@ def t_min(self):
         products result of the combination of multiple frames, min time must
         be the minimum of the start times
         """
-        return Time(self.get('t_min'), format='mjd')
+        t_min = self.get('t_min')
+        if np.isfinite(t_min):
+            return Time(t_min, format='mjd')
+        else:
```

Comment:
Wondering whether we should leak the `nan` here or just return `None`. @msdemlei does `nan` provide more meaning in this context?

Reply:
we already leak nan-s for many of the other elements, exptimes, etc. (It's only `Time` that chokes on them, Quantity handles it well, etc.)

Reply:
see the copy-pasted examples in https://github.com/astropy/pyvo/issues/460#issuecomment-1650572142

Reply:
OK. I've looked at that but I thought it might be from some representation inside `pyvo`